modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -0,0 +1,116 @@
+
+image::multiple-IdP-user-flow.png[User Flow for Configuring Multiple External Client Providers]
+
+=== Tasks for Implementing Multiple Client Providers Based on Roles
+
+Based on your role in your organization and your requirements, you might need to perform the following set of tasks to implement multiple client providers:
+
+[%header,cols="20a,40a,20a,50a"]
+|===
+| Role | Task | Where to Perform | Notes
+| Business Group Administrator | Assign multiple IdPs to an environment and also assign a default IdP. | Access Management | Using multiple IdPs provides flexibility to choose an IdP in API configuration inside API Manager, which can be useful when multiple teams are using the same environment for building APIs, but still want to use separate IdPs. This also helps in use cases when separate API instances are created for internal and external consumers, and separate IdPs based on a consumer is required. 
+| Business Group Administrator | Be notified about any possible interruption to API consumers when I am about to change any IdP configuration.  | Access Management | None
+| Master organization Administrator | Change the default IdP across all environments and across all business groups. | Access Management | Resetting the default IDP updates all environments across all business groups to a selected IDP, without affecting existing client applications. This way, when a new IdP is created for the organization, all environments are updated with just one change.
+| API Manager Administrator | Review which client provider an application belongs to on the client administrator page to be able to differentiate between different applications with the same name. | Access Management | None
+| API Owner | Configure my API in API Manager to automatically map to a default IdP configured by the business group administrator. I can also select a different IdP, if available | API Manager | None
+| API Owner | When promoting or importing an API between environments, specify an IdP from the target environment. | API Manager | If there is just one (default) IdP specified for the target environment, it is automatically assigned to the API on promotion. 
+| Application User | Request access to an API based on the API instance that exists in a specific environment. | Exchange | When you select the API for which you want to request access, you must first choose the API instance to which you are requesting access. The API instance is filtered based on the environment.
+|===
+
+=== Configure Multiple Client Providers
+
+To configure multiple external client providers:
+
+* Step 1: From *Access Management*, first enable xref:access-management::managing-api-clients.adoc[client management].
++
+As an Anypoint Administrator, you must first enable clients applications to be authenticated using OAuth for the client providers that you want to implement in your organization.
```

Comment:
Suggest omitting; redundant of the step above it. Otherwise...:
```suggestion
The Anypoint Administrator must first enable client applications to be authenticated using OAuth for the client providers to be implemented in the organization.
```

Reply:
Done! Thanks!